checker/tests/nullness/ReferencesDefaults.java

```diff
@@ -9,6 +9,7 @@ Object test() {
     }
 
     class Use {
+        // :: error: (assignment.type.incompatible)
         Decl d = new Decl();
```

Comment:
This one is also confusing... Is this because the `DefaultQualifier` on class `Decl` also applies to the class itself and then makes all `Decl` object creations nullable? We should think about that behavior...

Reply:
This error should not occur.